### PR TITLE
Dropping python 3.4 support

### LINKED ISSUE #733: Python3.4 deprecated?
From https://travis-ci.org/codelucas/newspaper/builds/575679321 that was associated with a PR I made, the Python3.4 build failed, due to the reason:
```
This lxml version requires Python 2.7, 3.5 or later.
```

### PR DESCRIPTION
**Pull request by Kerl1310**
Fixes #733

### CODE REVIEW

**Comment by Kerl1310** [comment]
Fixes build issues

**Comment by Kerl1310** [comment]
@codelucas Could you take a look at this PR? Fixes the issues with the build which would allow other changes to be made

File: requirements.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ PyYAML>=3.11
 requests>=2.10.0
 tinysegmenter==0.3  # TODO(codelucas): Investigate making this >=0.3
 tldextract>=2.0.1
+pandas>=0.25.3
```
Comment (codelucas) [inline requirements.txt:15]: why do this?

Comment (Kerl1310) [reply requirements.txt]: Mistakenly added, shall remove momentarily

File: requirements.txt [hunk 2]
```diff
@@ -11,4 +11,4 @@ python-dateutil>=2.5.3
 PyYAML>=3.11
 requests>=2.10.0
 tinysegmenter==0.3  # TODO(codelucas): Investigate making this >=0.3
-tldextract>=2.0.1
```
Comment (codelucas) [inline requirements.txt:14]: i wonder why this was marked as a diff? since there is no diff

**Pull request merged**